### PR TITLE
[TW-1541] Refresh Manage API deployments in Postman

### PR DESCRIPTION
**Pull request by akinard-postman**
Refresh includes:
- Minor revision to the opening statement.
- Rearranged the list of gateways based on their order in the UI.
- Updated the screenshot to match the current UI version.

### CODE REVIEW

**Review by petewitcher-postman — APPROVED** [review]
LGTM!

File: src/pages/docs/designing-and-developing-your-api/deploying-an-api/deploying-an-api-overview.md [hunk 1]
```diff
@@ -22,7 +22,9 @@ contextual_links:
     url: "https://blog.postman.com/the-new-postman-api-platform/"
 ---
 
-Stay on top of your API deployments in Postman by connecting to an API gateway. Postman integrates with popular API gateways, including [Amazon API Gateway](https://aws.amazon.com/api-gateway/), [Apigee X](https://cloud.google.com/apigee), and [Azure API Management](https://azure.microsoft.com/en-us/services/api-management/). Once connected to an API gateway, you can view your API deployment status and history from within Postman.
+Postman enables you to manage your API deployments by connecting to an API gateway. When you connect Postman to an API gateway, you can track your API deployment status and history.
+
+Postman offers integration with popular API gateways, such as [Apigee X](https://cloud.google.com/apigee), [Amazon API Gateway](https://aws.amazon.com/api-gateway/), and [Azure API Management](https://azure.microsoft.com/en-us/services/api-management/).
```
Comment (mark-anthony-postman) [inline src/pages/docs/designing-and-developing-your-api/deploying-an-api/deploying-an-api-overview.md:27]: Should we combine this sentence with the previous paragraph? That would create a featured snippet that's 49 words in length, which would be ideal per the [featured snippet guidelines](https://postmanlabs.atlassian.net/wiki/spaces/MPANS/pages/4493476409/Featured+Snippet+Guidelines).

**Review by mark-anthony-postman — APPROVED** [review]
LGTM!

File: src/pages/docs/designing-and-developing-your-api/deploying-an-api/deploying-an-api-overview.md (hunk 1 above)
Comment (akinard-postman) [reply src/pages/docs/designing-and-developing-your-api/deploying-an-api/deploying-an-api-overview.md:27]: It was that way originally so I can revert this update. 👍

**Review by jkonrath-postman — APPROVED** [review]
LGTM

**Pull request merged**